### PR TITLE
open-cluster-management/backlog#7582 Ad 21 upg addons

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Review by hanqiuzh — APPROVED** [review]

**Comment by hanqiuzh** [comment]
looks good to me 👍  thanks

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -75,6 +75,83 @@ This occurs when one or more ClusterImageSets on the existing version have the s
 
 . Restart the upgrade.
 
+[#upgrade-disables-klusterletaddonconfig]
+=== Upgrade to 2.1.1 disables klusterletaddonconfig CRDs
+
+// 2.1.1:7582
+
+When your {product-title-short} upgrades from version 2.1.0 to version 2.1.1, the klusterletaddonconfig custom resource definitions (CRDs) might be reinstalled during the upgrade. If this happens, all of the add ons show a `Disabled` status on the _Cluster settings_ pages. Complete the following steps to diagnose the problem and restore the klusterletaddonconfig CRDs:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: Update the following statement from: 

"When your {product-title-short} upgrades from version 2.1.0 to version 2.1.1, the klusterletaddonconfig custom resource definitions (CRDs) might be reinstalled during the upgrade."

TO 

"When your {product-title-short} upgrades from version 2.1.0 to  2.1.1, the `klusterletaddonconfig` custom resource definitions (CRDs) might be reinstalled during the upgrade."

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -75,6 +75,83 @@ This occurs when one or more ClusterImageSets on the existing version have the s
...
+
+. Log on to the hub cluster by using the `oc login` command:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: Are you prompting the user for the entire command? Or should the colon (:) be a period (.)?

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -75,6 +75,83 @@ This occurs when one or more ClusterImageSets on the existing version have the s
...
+
+. Run the following command to confirm that the deleted klusterletaddonconfig CRDs occurred because of the reinstalled CRDs:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: Change klusterletaddonconfig to `klusterletaddonconfig`

File: release_notes/known_issues.adoc [hunk 4]
```diff
@@ -75,6 +75,83 @@ This occurs when one or more ClusterImageSets on the existing version have the s
...
++
+----
+% oc get klusterletaddonconfig --all-namespaces 
+----
++
+If the returned content is `No resources found`, the reinstallation is likely the issue. Continue with step 3.
```
Comment (dockerymick) [inline release_notes/known_issues.adoc:93]: I like the way you worded this, I say "might" a lot lol

File: release_notes/known_issues.adoc [hunk 5]
```diff
@@ -75,6 +75,83 @@ This occurs when one or more ClusterImageSets on the existing version have the s
...
+EOF
+
+    fi
+    echo "  Done."
+done
+----
++
+Replace the value for `ACM_NAMESPACE` with the name of your namespace, if you did not install {product-title-short} in the `open-cluster-management` namespace.
+
+. Run the script from command line. Your command should resemble the following command:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: You can condense this, but not required. 

"Run the script from command line."  TO "Run the script from the CLI."

File: release_notes/known_issues.adoc [hunk 6]
```diff
@@ -75,6 +75,83 @@ This occurs when one or more ClusterImageSets on the existing version have the s
...
++
+----
+chmod +x restore-addons.sh && ./restore-addons.sh
+----
++
+Running the script recreates the missing klusterletaddonconfig CRDs in each managed cluster namespace.
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: "Running the script recreates the missing klusterletaddonconfig CRDs in each managed cluster namespace." 

TO 

"Running the script recreates the missing `klusterletaddonconfig` CRDs in each managed cluster namespace.

**Review by dockerymick — APPROVED** [review]
Made a few suggestions, otherwise it all looks good. Numbers and alignment look good

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Should be a period. Good catch. Change in approach.

File: release_notes/known_issues.adoc (hunk 4 above)
Comment (cadawson) [reply release_notes/known_issues.adoc:93]: Thanks. Either one works, and I use them both.

**Pull request merged**